201-front-door-standard-premium-app-service-public/main.bicep

```diff
@@ -0,0 +1,175 @@
+param location string = resourceGroup().location
+param appName string
+param appServicePlanSkuName string
+param appServicePlanCapacity int
+param frontDoorEndpointName string

```

Comment:
add a defaultValue using uniqueSTring (if this needs to be unique, otherwise a literal is fine)

Reply:
Done.